### PR TITLE
Feat/drag tasks

### PR DESCRIPTION
**Pull request by ekertt**

### CODE REVIEW

File: src/api/tasks.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Task } from '../models/Task';
 import { CreateTask } from '../models/Task';
 
-const baseUrl = 'https://tic2b5chf5.execute-api.eu-west-1.amazonaws.com/prod';
+const baseUrl = 'https://pk2kqkbu6l.execute-api.eu-west-1.amazonaws.com/prod';
```
Comment (Ryukishi) [inline src/api/tasks.ts:4]: should these not be secret variables in the env?

File: src/awsconfig.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "Auth": {
     "region": "eu-west-1",
-    "userPoolId": "eu-west-1_ESOmKrJEH",
-    "userPoolWebClientId": "3d5i943ditllju4upg6rsmikk",
+    "userPoolId": "eu-west-1_EvjxRSWkg",
+    "userPoolWebClientId": "63pt4gqlvh725e9592209phluk",
```
Comment (Ryukishi) [inline src/awsconfig.json:5]: secret variable in env?

File: src/views/NotAuthenticated.tsx
Comment (Ryukishi) [inline src/views/NotAuthenticated.tsx:1]: We should not forget to refactor this later on

Comment (ekertt) [reply src/views/NotAuthenticated.tsx:1]: Yeah that's true, same for the env variables for now. Since this is just for testing purpose!

**Pull request merged**